lib/SIL/SILDeclRef.cpp

```diff
@@ -579,6 +579,11 @@ IsSerialized_t SILDeclRef::isSerialized() const {
   if (dc->getResilienceExpansion() == ResilienceExpansion::Minimal)
     return IsSerialized;
 
+  // If module is fragile, everything inside it is fragile.
+  if (dc->getParentModule()->getResilienceStrategy() ==
```

Comment:
What is the motivation for this change? It makes the SIL verifier assertions ineffective. Recall that if Sema fails to diagnose an invalid usage of a function from an inlineable function, the SIL verifier then catches the reference. But now, you're saying everything is marked 'serialized', so the checks will not take place.